added identity_ids check when type=UserAssigned

BerendvW opened the pull request:
When an Managed instance is of type = "UserAssigned" the identity block for webapp slot, doesn't differentiate between the types.
this change makes it that if reads from the right variable if given type UserAssigned

BerendvW committed:
added identity_ids check when type=UserAssigned

When an Managed instance is of type = "UserAssigned" the identity block for slot, doesn't differentiate between the types.
this change makes it that if reads from the right variable if given type UserAssigned

BerendvW committed:
managed.identities in stead of identity_ids

dimitrifc reviewed (approved):
LGTM, make a PR upstream as well https://github.com/aztfmod/terraform-azurerm-caf/pulls

BerendvW committed:
added  key_vault_reference_identity_id

 key_vault_reference_identity_id was missing.
copied from function app where it is present

BerendvW committed:
added key_vault_reference_identity_id to slot

BerendvW committed:
Update slot.tf

BerendvW committed:
Update slot.tf
